src/ui/components/results/universalresultscomponent.js

```diff
@@ -51,11 +51,24 @@ export default class UniversalResultsComponent extends Component {
     return true;
   }
 
+  mount (container) {
+    const searchState = this.getState('searchState');
+    const camelCase = (str) => {
```

Comment:
can we make this a general util function and add jsdocs to it? it might be too specific if it does the `-` splitting but can take in an array of words to camel case. i believe we have a strings.js util file

Reply:
We don't have a strings.js file but I can make one